app/controllers/giveaways_controller.rb

```diff
@@ -55,10 +55,6 @@ def edit
     def show
         @giveaway = Giveaway.find(params[:id])
         if !@giveaway.isActive
```

Comment:
nitpick on Rails conventions: use `Giveaway#active?` instead of `Giveaway#isActive`